app/src/main/java/org/oppia/android/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -170,8 +169,25 @@ class ProfileChooserFragmentPresenter @Inject constructor(
       val scrollableWidth = binding.profilesListLandscape?.let {
         it.width - (it.paddingStart + it.paddingEnd)
       } ?: 0
-      val offset =
-        if (isLeft) scrollDistance - scrollableWidth else scrollableWidth - scrollDistance
+
+      // Check layout direction.
+      val isRtl = binding.profilesListLandscape?.let { ViewCompat.getLayoutDirection(it) } ==
+        ViewCompat.LAYOUT_DIRECTION_RTL
+
+      val scrollLeftInRtl = isRtl && isLeft
+      val scrollRightInRtl = isRtl && !isLeft
+      val scrollLeftInLtr = !isRtl && isLeft
+      val scrollRightInLtr = !isRtl && !isLeft
```

Comment:
Are there tests for verifying this RTL/LTR and directional scrolling behaviors?

Reply:
I have added tests that verify both conditions when using the arrows. Swipe behaviour is covered by existing tests using "scrollTo()".